### PR TITLE
See generated code in community/enterprise files when building

### PR DESCRIPTION
**Pull request by chloegugli**

### CODE REVIEW

**Comment by chloegugli** [comment]
There has been a regression in commit "Refactor the project (#3)", and we couldn't see the code generated by "grunt replace" in templates_version/community and enterprise files. It made this files out of date and this could cause misunderstanding.

File: Gruntfile.js [hunk 1]
```diff
@@ -324,7 +340,7 @@ module.exports = function (grunt) {
                     {
                         flatten: true,
                         src: ['<%= inspinia.app %>/templates_versions/<%= inspinia.version %>/config.js'],
-                        dest: '<%= inspinia.app %>/scripts/config.js'
```
Comment (paraita) [inline Gruntfile.js]: Can you try removing this part ? the src and dest are the same

File: Gruntfile.js [hunk 2]
```diff
@@ -335,12 +351,12 @@ module.exports = function (grunt) {
                     {
                         flatten: true,
                         src: ['<%= inspinia.app %>/templates_versions/<%= inspinia.version %>/app.js'],
-                        dest: '<%= inspinia.app %>/scripts/app.js'
```
Comment (paraita) [inline Gruntfile.js]: Can you try removing this part ? the src and dest are the same

File: Gruntfile.js [hunk 3]
```diff
@@ -335,12 +351,12 @@ module.exports = function (grunt) {
...
+                        dest: '<%= inspinia.app %>/templates_versions/<%= inspinia.version %>/app.js'
                     },
                     {
                         flatten: true,
                         src: ['<%= inspinia.app %>/templates_versions/<%= inspinia.version %>/index.html'],
-                        dest: '<%= inspinia.app %>/index.html'
```
Comment (paraita) [inline Gruntfile.js]: Can you try removing this part ? the src and dest are the same

**Pull request closed without merging**